lib/bolt/target.rb

```diff
@@ -40,6 +41,14 @@ def initialize(uri, options = nil)
       if @options['protocol']
         @protocol = @options['protocol']
       end
+
+      if @options['host']
+        @host = @options['host']
+      end
+
+      # WARNING: name should never be updated
+      @name = @options['name'] || @uri
+      # TODO: this should error unless name is set
```

Comment:
Do you want to leave this in?

Reply:
I'm removing it. I don't think it matters until we update `Target` in puppet.